### PR TITLE
Support Databricks 11.3 [databricks]

### PR DESCRIPTION
**Pull request by NvTimLiu**
Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
CI script for issue: https://github.com/NVIDIA/spark-rapids/issues/7352

File: jenkins/Jenkinsfile-blossom.premerge [hunk 1]
```diff
@@ -381,6 +387,39 @@ pipeline {
...
+                        BASE_SPARK_VERSION = "${RUNTIME_MAP["$DB_RUNTIME"][ID_SPARK]}"
+                        BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS = "${RUNTIME_MAP["$DB_RUNTIME"][ID_INSTALL]}"
+                        INIT_SCRIPTS = getInitScripts("$INIT_SCRIPTS_DIR",
+                                "${RUNTIME_MAP["$DB_RUNTIME"][ID_INITSCRIPTS]}")
+                    }
+                    steps {
+                        script {
+                            timeout(time: 5, unit: 'HOURS') {
+                                unstash "source_tree"
+                                databricksBuild()
```
[the remark this answers is not in the document]
Comment (pxLi) [reply jenkins/Jenkinsfile-blossom.premerge:417]: @NvTimLiu 
could add condition to skip test stage if DB_RUNTIME == 11.3 https://github.com/NVIDIA/spark-rapids/blob/branch-23.02/jenkins/Jenkinsfile-blossom.premerge#L473
and also a TODO to remind us to re-enable after all tests get working as expected, thanks~

[the remark this answers is not in the document]
Comment (NvTimLiu) [reply jenkins/Jenkinsfile-blossom.premerge:417]: Fixed, skip test phase for 11.3

**Comment by NvTimLiu** [comment]
build

**Review by pxLi — APPROVED** [review]

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
Lets merge this for 330db build check in premerge CI, @NvTimLiu will help setup nightly CI for 330db later

**Pull request merged**